src/azure-cli/azure/cli/command_modules/sql/custom.py

```diff
@@ -3343,6 +3344,9 @@ def elastic_pool_update(
         cmd.cli_ctx,
         maintenance_configuration_id)
 
+    if high_availability_replica_count:
```

Comment:
```suggestion
    if high_availability_replica_count is not None:
```
In case customer wants to update the number with 0